app/views/shared/_api_selector.html.slim

```diff
@@ -1,7 +1,8 @@
 = javascript_pack_tag 'api_selector'
+- master = current_account.master?
```

Comment:
Maybe `is_master` to make it clear that the variable holds a boolean and not the actual master account object.